docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-iis-installation-powershell.md

```diff
@@ -0,0 +1,417 @@
+   authentication process.
+
+               a. `SigningKey:` must be a Base64-encoded string
+               b. `Authority and IssuerUrl:` should be the same URL as your application
+               c. `AllowRegistration:` to true allows unrestricted registration of new Admin API clients. This is not recommended for production.
+
+     4. Configure `$p`. This is the variable used to send all the information to
+     the installation process.
+
+1. 1. `ToolsPath`. Path for storing installation tools, e.g., nuget.exe.
+      Defaults to "C:/temp/tools"
+   2. **`OdsApiUrl`. Base URL for the ODS / API. Mandatory.**
+   3. `PackageVersion`. Optional. If not set, will retrieve the latest full
+      release package.
+
+> [!NOTE] ![Screenshot](https://edfi.atlassian.net/wiki/images/icons/grey_arrow_down.png)
+>
+> SQL Server
+>
+> ```
+> $dbConnectionInfo = @{
+>         Server = "(local)"
+>         Engine = "SqlServer"
+>         UseIntegratedSecurity = $false
+>         Username = "exampleAdmin"
+>         Password = "examplePassword"
+> }
+> $adminApiFeatures = @{
+>     ApiMode = "sharedinstance"
+> }
+> $authenticationSettings = @{
+>     Authority = "[https://localhost/adminapi](https://localhost/adminapi)"
+>     IssuerUrl = ""[https://localhost/adminapi](https://localhost/adminapi)"
+>     SigningKey = "Base64-encoded string"
+>     AllowRegistration = $false
+> }
+> $p = @{
+>     ToolsPath = "C:/temp/tools"
+>     DbConnectionInfo = $dbConnectionInfo
+>     OdsApiUrl = "[http://web-api.example.com/WebApi](http://web-api.example.com/WebApi)"
+>     PackageVersion = '1.4.0.0'
+>     PackageSource = $adminApiSource
+>     AuthenticationSettings = $authenticationSettings
+>     AdminApiFeatures = $adminApiFeatures
+> }
+> ```
+>
+> ![Screenshot](https://edfi.atlassian.net/wiki/images/icons/grey_arrow_down.png)
+>
+> PostgreSQL Server
+>
+> ```
+> $dbConnectionInfo = @{
+>         Server = "localhost"
+>         Engine = "PostgreSQL"
+>         UseIntegratedSecurity = $false
+>         Username = "postgres"
+>         Password = "examplePassword"
+> }
+> $adminApiFeatures = @{
+>     ApiMode = "sharedinstance"
+> }
+> $authenticationSettings = @{
+>     Authority = "[https://localhost/adminapi](https://localhost/adminapi)"
+>     IssuerUrl = ""[https://localhost/adminapi](https://localhost/adminapi)"
+>     SigningKey = "Base64-encoded string"
+>     AllowRegistration = $false
+> }
+> $p = @{
+>     ToolsPath = "C:/temp/tools"
+>     DbConnectionInfo = $dbConnectionInfo
+>     OdsApiUrl = "[http://web-api.example.com/WebApi](http://web-api.example.com/WebApi)"
+>     PackageVersion = '1.4.0.0'
+>     PackageSource = $adminApiSource
+>     AuthenticationSettings = $authenticationSettings
+>     AdminApiFeatures = $adminApiFeatures
+> }
+> ```
+
+### **Step 3. Open a PowerShell Prompt in Administrator Mode**
+
+Method 1: Open \[Windows Key\]-R which will open a Run dialog for tasks needing
```

Comment:
In the ODS/API documentation I found a number of examples mentioning the Windows key. I started using this emoji: 🪟.